site/src/components/progress-bar.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
This page should also be added to the accessibility testing list in integration/tests/accessibility.spec.ts